packages/forklift-console-plugin/src/modules/ProvidersNG/modals/EditProviderDefaultTransferNetwork/EditProviderDefaultTransferNetwork.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Modify, V1beta1Provider } from '@kubev2v/types';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/core-api';
+
+import { EditModalProps } from '../EditModal';
+
+import { OpenshiftEditProviderDefaultTransferNetwork } from './OpenshiftEditProviderDefaultTransferNetwork';
+
+export type EditProviderDefaultTransferNetworkProps = Modify<
+  EditModalProps,
+  {
+    resource: V1beta1Provider;
+    title?: string;
+    label?: string;
+    model?: K8sModel;
+    jsonPath?: string | string[];
+  }
+>;
+
+export const EditProviderDefaultTransferNetwork: React.FC<
+  EditProviderDefaultTransferNetworkProps
+> = (props) => {
+  switch (props.resource?.spec?.type) {
```

Comment:
EditProviderDefaultTransferNetwork is only openshift, no need for general purpose method